articles/azure-resource-manager/management/manage-resource-groups-portal.md

```diff
@@ -16,7 +16,13 @@ Learn how to use the [Azure portal](https://portal.azure.com) with [Azure Resour
 
 A resource group is a container that holds related resources for an Azure solution. The resource group can include all the resources for the solution, or only those resources that you want to manage as a group. You decide how you want to allocate resources to resource groups based on what makes the most sense for your organization. Generally, add resources that share the same lifecycle to the same resource group so you can easily deploy, update, and delete them as a group.
 
-The resource group stores metadata about the resources. Therefore, when you specify a location for the resource group, you are specifying where that metadata is stored. For compliance reasons, you may need to ensure that your data is stored in a particular region. Note that resources inside a resource group can be of different regions.
+The resource group scope is also used throughout the Azure Portal to create views that spans across multiple resources, for example:
+- Metrics blade provides metrics information (CPU, resources) to users. 
+- Deployments blade shows the history of ARM Template or Bicep deployments targeted to that Resource Group (which includes Portal deployments).
+- Policy blade provides information related to the policies enforced on the resource group.
+- Diagnostics settings blade provides the ability to diagnose errors or review warnings. 
+
+Therefore, when you specify a location for the resource group, you are specifying where that metadata is stored. For compliance reasons, you may need to ensure that your data is stored in a particular region. Note that resources inside a resource group can be of different regions. 
```

Comment:
It seems that since the file exists on your fork I do not have permissions to make any edits to the file.  
Additional feedback:
- Add back in "_The resource group stores metadata about the resources._ " before "_Therefore, when you specify a location..._"
- Fix verb agreement for the newly added sentence: "_...create views that **span** across multiple resources..._"

Reply:
Thanks. Done. I wouldn't worry about grammar because @Court72 has editing access and does a great job of fixing grammar before merging PRs.